src/NServiceBus.AcceptanceTests/Routing/When_configuring_routing.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+
+    public class When_configuring_routing : NServiceBusAcceptanceTest
```

Comment:
I went for ATTs becasue it shows me the message user will see too. But now you pointed it out, maybe that's overcomplicating things.